source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB specific information to users,
+supplemented by links to MongoDB Documentation. The |copilot| is
+knowledgeable of the latest MongoDB Documentation and uses the
+MongoDB Docs Chatbot API to link users to information in addition to
+the textual response. 
+
+Example
+~~~~~~~
+
+Consider the following prompt that asks for information on creating an
+index in the GitHub Copilot chat:
+
+.. code-block:: javascript
```

Comment:
To avoid the weird syntax highlighting in the code blocks, maybe try `.. code-block:: none` here. Same comment applies to the response code-block further down.

Reply:
Thanks for this suggestion! I also updated the /query page to use the none type to avoid the weird syntax highlighting